pkg/agent/agent_windows.go

```diff
@@ -120,8 +135,10 @@ func (i *Initializer) prepareOVSBridge() error {
 		if err := i.ovsBridgeClient.Delete(); err != nil {
 			klog.Errorf("Failed to delete OVS bridge: %v", err)
 		}
-		if err := util.DeleteHNSNetwork(util.LocalHNSNetwork); err != nil {
-			klog.Errorf("Failed to cleanup host networking: %v", err)
+		if !i.enableExternalNode {
```

Comment:
According to line 116, this cannot happen here?

Reply:
removed.